src/modules/recipes/models/Area.ts

```diff
@@ -0,0 +1,31 @@
+export enum Areas {
+  American = "American",
+  British = "British",
+  Canadian = "Canadian",
+  Chinese = "Chinese",
+  Croatian = "Croatian",
+  Dutch = "Dutch",
+  Egyptian = "Egyptian",
+  Filipino = "Filipino",
+  French = "French",
+  Greek = "Greek",
+  Indian = "Indian",
+  Irish = "Irish",
+  Italian = "Italian",
+  Jamaican = "Jamaican",
+  Japanese = "Japanese",
+  Kenyan = "Kenyan",
+  Malaysian = "Malaysian",
+  Mexican = "Mexican",
+  Moroccan = "Moroccan",
+  Polish = "Polish",
+  Portuguese = "Portuguese",
+  Russian = "Russian",
+  Spanish = "Spanish",
+  Thai = "Thai",
+  Tunisian = "Tunisian",
+  Turkish = "Turkish",
+  Ukrainian = "Ukrainian",
+  Unknown = "Unknown",
+  Vietnamese = "Vietnamese",
+}
```

Comment:
It's useful to add an array of object values for enums when you expect to iterate through values of enum (e.g. if you want to create Select form element).
```suggestion
}
export const Areas = Object.values(Area);
```